### PR TITLE
Updated managedservices module to support api-version 2019-06-01

### PR DESCRIPTION
**Pull request by rajivnandivada**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by rajivnandivada** [comment]
@tjprescott @limingu @yugangw-msft could you please review this?

File: src/azure-cli/azure/cli/command_modules/managedservices/custom.py [hunk 1]
```diff
@@ -50,11 +44,11 @@ def cli_definition_create(cmd, client,
         Plan=plan,
     )
 
-    return client.create_or_update(
+    poller = client.create_or_update(
```
Comment (yugangw-msft) [inline src/azure-cli/azure/cli/command_modules/managedservices/custom.py]: any reason you don't return `poller` here? CLI core has the common logic to detect the case that if it is a poller and will wait to get the result

Comment (rajivnandivada) [reply src/azure-cli/azure/cli/command_modules/managedservices/custom.py]: I didnt know that. I have updated all such references to just return the poller.

**Comment by panchagnula** [comment]
BTW update history as well - this file has moved here now ~src/azure-cli/HISTORY.rst

**Pull request merged**